scripts/form-validation.js

```diff
@@ -0,0 +1,20 @@
+function validateEmail() {
+  const email = document.getElementById('email').value;
```

Comment:
- [ ] [OPTIONAL] To avoid side-effects I think it would be better if you passed in the value `email` into the `validateEmail` function rather than querying it inside the function.